src/features/submission/components/submission-form/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { UploadOutlined } from '@ant-design/icons';
+import { Button, Form, FormInstance, InputNumber, Upload } from 'antd';
+import React from 'react';
+
+const SubmissionForm = ({ form }: { form: FormInstance }) => {
+  const uploadProps = {
+    beforeUpload: () => {
```

Comment:
I noticed this also in the profile upload - what is it used for?

Reply:
instance of the form is available to the outer component, cuz i'm using the modal's submit button instead of the form's so the modal has access to the form's values